tests/functional/testTutorials.js

```diff
@@ -27,8 +29,23 @@ describe('Tutorials', function () {
     await vpn.clickOnElement(homeScreen.TUTORIAL_LEAVE);
   }
 
-  describe('Tutorial tooltip', function () {
+  async function loadAddons() {
+    await vpn.waitForElementProperty(
+        'VPNAddonManager', 'loadCompleted', 'true');
+    vpn.resetLastAddonLoadingCompleted();
+    await vpn.setSetting(
+        'addon/customServerAddress',
+        `http://localhost:${constants.ADDON_PORT}/04_tutorials_basic/`);
+    await vpn.setSetting('addon/customServer', 'true');
+    await vpn.resetAddons();
+    await vpn.waitForCondition(async () => {
+      return await vpn.lastAddonLoadingCompleted();
```

Comment:
No `await` needed this doesn't return a promise.

Reply:
Can we just change `resetAddons` to return this condition everytime? This way that promise will only resolve once the reset is complete and also a bit of the complexity will be hidden from the caller.